### PR TITLE
[FIX] website_forum: modify notification message

### PR DESCRIPTION
**Pull request by vivp-odoo**
**Before PR:**
Currently, when a user marks their answer as the best answer in the forum, an error notification appears with the message 'Sorry, you cannot vote for your own posts' However, this error message lacks clarity and does not clarify the underlying issue.

**After PR:**
Now the error message is modified to 'Sorry,
you cannot select your own posts as the best answer'

**Task**-3481264

### CODE REVIEW

**Review by julien-banken — APPROVED** [review]

File: addons/website_forum/i18n/website_forum.pot [hunk 1]
```diff
@@ -2991,6 +2991,12 @@ msgstr ""
 #. module: website_forum
 #. odoo-javascript
 #: code:addons/website_forum/static/src/js/website_forum.js:0
```
Comment (std-odoo) [inline addons/website_forum/i18n/website_forum.pot:2993]: in the commit message
> Sorry, you cannot vote for your own post**s**

**Review by std-odoo — COMMENTED** [review]
@vivp-odoo Hi :) 

One small comment about the commit message, otherwise LGTM

Cheers

File: addons/website_forum/i18n/website_forum.pot (hunk 1 above)
Comment (vivp-odoo) [reply addons/website_forum/i18n/website_forum.pot:2993]: Hello @std-odoo 

I have made the changes :+1:

**Comment by std-odoo** [comment]
@robodoo r+

**Pull request closed without merging**